CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+**v2.0.0:**
+- added integration with [official Databricks extensions](https://marketplace.visualstudio.com/items?itemName=databricks.databricks)
+  - new connection manager [Databricks Extensions](README.md/#setup-and-configuration-databricks-extension-connection-manager)
+  - derive cluster for [SQL Browser](README.md/#sql-browser)
+  - change cluster using [Cluster Manager](README.md/#cluster-manager)
+  - added File System `wsfs:/` to replace `dbws:/` in the future (currently both are still supported)
```

Comment:
Do you want to mention that the notebooks support also picks the cluster from the extension by default?